src/client/entity-editor/relationship-editor/types.ts

```diff
@@ -64,3 +64,75 @@ export type LanguageOption = {
 	name: string,
 	id: number
 };
+
+export enum RelationshipTypes {
+	AuthorWorkedOnWork = 1,
+	AuthorIllustratedEdition = 2,
+	EditionIsAnEditionOfEditionGroup = 3,
+	PublisherPublishedEdition = 4,
+	AuthorEditedEdition = 5,
+	WorkInspiredWork = 6,
+	WorkIsParodyOfWork = 7,
+	AuthorWroteWork = 8,
+	AuthorTranslatedWork = 9,
+	EditionContainsWork = 10,
+	AuthorMarriedToAuthor = 11,
+	AuthorInvolvedWithAuthor = 12,
+	AuthorMemberOfGroup = 13,
+	AuthorParentOfAuthor = 14,
+	AuthorSiblingOfAuthor = 15,
+	SubgroupOfGroup = 16,
+	AuthorFounderOfGroup = 17,
+	PenNameOfAuthor = 18,
+	AuthorIsCollaboratorOnShortTermProject = 19,
+	AuthorWasFounderOfPublisher = 20,
```

Comment:
```suggestion
	AuthorIsFounderOfPublisher = 20,
```

I think otherwise one might assume that there are different relationships for is/was, as you use "is"  everywhere else